src/test/java/tests/ValidLoginTest.java

```diff
@@ -0,0 +1,40 @@
+package tests;
+
+import base.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.RegistrationPage;
+import pages.ValidationPage;
+import utilities.LoadProperties;
+
+public class ValidLoginTest extends TestBase {
+
+    HomePage homePageObj;
+    RegistrationPage registrationPageObj;
+    ValidationPage validationPageObj;
+    LoginPage loginObj;
+    String validEmail = LoadProperties.userData.getProperty("Email");
+    String validPassword = LoadProperties.userData.getProperty("Password");
+
+    @BeforeMethod
+    public void beforeTest() {
+        homePageObj = new HomePage(driver);
+        validationPageObj = new ValidationPage(driver);
+        loginObj = new LoginPage(driver);
+    }
+
+    @Test(priority = 0)
+    public void clickOnLogin(){
+        homePageObj.clickOnMyAccountButton()
+                .clickOnLoginButton();
+    }
+
+    @Test(priority = 1)
```

Comment:
this test should depend on the previous test, you should but something like this 
@Test(priority = 1, dependsOnMethods = { "clickOnLogin" })

by doing this dependency if the first method "clickOnLogin " failed the second method "userValidLogin " will be skipped and not failed

and that's the correct expectations.. if selenium wasn't able to click on login button then the webdriver will not navigate to the login page and the webelements in login page will not be displayed 

so you need to skip this method if the login button failed and not to fail it as actually it didn't fail it's just was not navigated to the correct screen